sdks/python/apache_beam/runners/interactive/cache_manager.py

```diff
@@ -107,6 +127,13 @@ def __init__(self, cache_dir=None):
       self._cache_dir = tempfile.mkdtemp(
           prefix='interactive-temp-', dir=os.environ.get('TEST_TMPDIR', None))
     self._versions = collections.defaultdict(lambda: self._CacheVersion())
+
+    if cache_format not in self._available_formats:
+      raise ValueError("Unsupported cache format: '%s'." % cache_format)
+    self._Reader, self._Writer = self._available_formats[cache_format]
```

Comment:
Any specific reason to name them as _Reader / _Writer? It's probably more common to name private variables as _reader / _writer.

Ref: Google Style Guide http://google.github.io/styleguide/pyguide.html
PEP8 Style Guide https://www.python.org/dev/peps/pep-0008/#naming-conventions

Reply:
Yes, I agree. I just wanted to indicate that those variables refer to class objects rather than functions, and to distinguish them from `iobase.NativeSource.reader`, `iobase.NativeSink.writer`, etc.

I renamed the variables as `self._reader_class` and `self._writer_class`, which should follow the naming convention but also let the user know that those are not functions?